src/include/daos_task.h

```diff
@@ -292,6 +293,11 @@ typedef struct {
 	daos_prop_t		*prop;
 } daos_cont_query_t;
 
+typedef struct {
```

Comment:
(style)  do not add new typedefs

Reply:
Ignoring this one to be consistent with the rest of the structs in this file.